api/python/t4/api.py

```diff
@@ -101,6 +102,44 @@ def delete(target):
     delete_object(bucket, path)
 
 
+def delete_package(name, registry=None):
+    if not re.match(PACKAGE_NAME_FORMAT, name):
+        raise QuiltException("Invalid package name, must contain exactly one /.")
```

Comment:
the RE could fail for other reasons than `/`, right?

Reply:
I don't think so. This check is a direct carry-over of `Package.validate_package_name`. It'd actually be better to move `validate_package_name` to `utils`, though, and just use that method here. TODO.

Reply:
Backlogged this method move. It's better addressed in its own PR as it's pretty merge conflict-y.